### PR TITLE
Fix overflowing elements that causes strange horizontal panning on mobile

### PR DESCRIPTION
**Pull request by mmmavis**
Related PRs/issues: #10232 

---

The root cause is due to mismatched layout class combo - for example, a Bootstrap container class (`container`) is paired with Tailwind row class (`tw-row`). Bootstrap uses 15px for layout paddings and margin offsets while Tailwind uses 1rem/16px. Mix and match the combo will create tiny misalignment or overflow that are hard to spot with bare eyes.

**Percy is expected to flag tiny visual difference**

--- 

Using homepage's `.section-take-action` section as an example:

image

image

- 1st screencap: `tw-row` div has width: 377px
- 2nd screencap: ancestor `container` div from two layer above has width 345+15+15=375px
- child div `tw-row` is wider by 2px which then creates overflow.

---

Note that I've only fixed areas where I was able to spot the issue or thought the Bootstrap -> Tailwind conversion would help. If we find more issues we can always create new tickets/PRs for them. We still have a long way to fully transition everything from Bootstrap to Tailwind 😓 

## To QA

Load the following links on your iPhone Safari

### Homepage
  - review app: https://foundation-s-10232-over-x4vpvk.herokuapp.com
  - prod: https://foundation.mozilla.org
 
### Primary page

Narrow layout
  - review app: https://foundation-s-10232-over-x4vpvk.mofostaging.net/who-we-are/
  - prod: https://foundation.mozilla.org/who-we-are/

Wide layout
  - review app: https://foundation-s-10232-over-x4vpvk.mofostaging.net/primary-page-test-wide-layout/
  - prod: https://foundation.mozilla.org/advocacy/

### Article page

  - review app: https://foundation-s-10232-over-x4vpvk.mofostaging.net/publication-page-with-chapter-pages/fixed-title-chapter-page/fixed-title-article-page/
  - prod: https://foundation.mozilla.org/docs/how-do-i-wagtail/grimms-fairy-tales-demo/wolf-and-seven-little-kids/

### Opportunity page 
  - review app: https://foundation-s-10232-over-x4vpvk.mofostaging.net/opportunity-page-test/
  - prod: https://foundation.mozilla.org/initiatives/web-literacy/core-curriculum/participate/

### CODE REVIEW

File: source/sass/wagtail/blocks.scss [hunk 1]
```diff
@@ -13,22 +13,22 @@
 .primary-page-content {
   &.wide {
     .streamfield-content {
-      @extend .col-12;
+      @apply tw-relative tw-w-full tw-px-8 tw-flex-grow-0 tw-flex-shrink-0 tw-basis-full tw-max-w-full;
```
Comment (mmmavis) [inline source/sass/wagtail/blocks.scss:16]: I basically converted all CSS properties Boostrap's `.col-12` has into Tailwind classes.

File: source/sass/wagtail/blocks.scss [hunk 2]
```diff
@@ -13,22 +13,22 @@
...
     }
   }
 
   &.narrow {
     .streamfield-content {
-      @extend .col-md-8;
+      @apply tw-relative tw-w-full tw-px-8 medium:tw-flex-grow-0 medium:tw-flex-shrink-0 medium:tw-basis-8/12 medium:tw-max-w-[8/12];
```
Comment (mmmavis) [inline source/sass/wagtail/blocks.scss:22]: likewise

File: source/sass/wagtail/blocks.scss [hunk 3]
```diff
@@ -13,22 +13,22 @@
...
     }
   }
 }
 
 .article-page-content {
   .streamfield-content {
-    @extend .col-md-10;
+    @apply tw-relative tw-w-full tw-px-8 medium:tw-flex-grow-0 medium:tw-flex-shrink-0 medium:tw-basis-10/12 medium:tw-max-w-[10/12];
```
Comment (mmmavis) [inline source/sass/wagtail/blocks.scss:29]: likewise

File: source/sass/wagtail/blocks.scss [hunk 4]
```diff
@@ -13,22 +13,22 @@
...
     &.wide {
-      @extend .col-12;
+      @apply tw-relative tw-w-full tw-px-8 tw-flex-grow-0 tw-flex-shrink-0 tw-basis-full tw-max-w-full;
```
Comment (mmmavis) [inline source/sass/wagtail/blocks.scss:31]: likewise

File: source/sass/wagtail/blocks.scss [hunk 5]
```diff
@@ -37,19 +37,18 @@
   .streamfield-content {
     @extend .col-lg-8;
     &.wide {
-      @extend .col-12;
+      @apply tw-relative tw-w-full tw-px-8 tw-flex-grow-0 tw-flex-shrink-0 tw-basis-full tw-max-w-full;
```
Comment (mmmavis) [inline source/sass/wagtail/blocks.scss:40]: likewise

File: source/sass/wagtail/blocks.scss [hunk 6]
```diff
@@ -37,19 +37,18 @@
...
     }
     &.full-width {
-      @extend .col-12;
+      @apply tw-relative tw-flex-grow-0 tw-flex-shrink-0 tw-basis-full;
```
Comment (mmmavis) [inline source/sass/wagtail/blocks.scss:43]: This one is a special case. 

I converted all CSS properties Boostrap's `.col-12` has into Tailwind classes and removed the ones that would override our custom `.full-width` rules.

**Review by danielfmiranda — APPROVED** [review]
Thanks @mmmavis! 

I just tested all the links provided below and can confirm that on the production site I am able to reproduce the issue, and on the review app the issue is fixed 👍 approved!

**Pull request merged**